web/views/layout.hbs

```diff
@@ -5,16 +5,55 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Sabka Bazar</title>
+    <title>{{pageName}} Page</title>
     <link href="/css/{{pageName}}.css" rel="stylesheet">
 </head>
 
 <body>
     {{> header}}
     {{{body}}}
     {{> footer}}
+    <!-- The Modal -->
+    <!-- The Modal -->
+    <div id="cart-modal" class="modal">
+        <!-- Modal content -->
+        <div class="modal-content">
+            <div class="modal-header">
+                <span class="close">&times;</span>
```

Comment:
Why this span is used and not Button, never use such tags for clickable(intractable) item. This is a very bad practice for accessibility